legacy/skills/src/main/res/values/dimens.xml

```diff
@@ -8,4 +8,5 @@
   <dimen name="payment_methods_width">340dp</dimen>
   <dimen name="payment_methods_width_landscape">544dp</dimen>
   <dimen name="onboarding_width_landscape">600dp</dimen>
+  <dimen name="waiting_for_players_width_landscape">660dp</dimen>
```

Comment:
why 660dp and not 600dp? 600dp is bigger than 99% of the devices in portrait mode. I think we should have just one number for that.